2-mandatory/8-journey-planner.js

```diff
@@ -18,8 +18,15 @@
   When you finish the exercise, think about how this solution is different to your last solution.
   What's better about each approach?
 */
+function journeyPlanner(locations, transportMode) {
+  let canTravelTo = [];
+  for (key in locations) {
+    let location = key;
```

Comment:
All this does is rename the variable. Maybe better to store `location=locations[key]`

Reply:
I've been doing this multiple times for this repo thinking that it makes my code more readable, I guess it clutters it more than anything else.  Will try to avoid doing this in the future.